tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/internal/findandreplace/FindReplaceLogicTest.java

```diff
@@ -474,6 +478,43 @@ public void testWholeWordSearchAvailable() {
 		assertThat(findReplaceLogic.isWholeWordSearchAvailable(""), is(false));
 	}
 
+	@Test
+	public void testReplaceInScopeStaysInScope() {
+		String lineSeparator= System.lineSeparator();
```

Comment:
You can use a static import and then directly use `lineSeparator()`.

Reply:
Yes, done in https://github.com/eclipse-platform/eclipse.platform.ui/commit/8e8a527bb19dad39d15c797d7cf57a8d889b579b